mozetl/search/search_rollups.py

```diff
@@ -45,6 +55,9 @@ def explode_searches(dataframe):
         dataframe
         .withColumn("search_counts", F.explode("search_counts"))
         .select(columns + exploded_search_columns)
+        .where("search_count > -1 or search_count is null")
```

Comment:
Please update the comment above to note that we're both exploding search counts and filtering out records from sources outside the above whitelist.